### PR TITLE
Unpin librosa and update SciPy pin

### PR DESCRIPTION
**Pull request by peterbell10**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #72834
* #72833

This removes the upper bound to librosa's pin and updates the scipy
pin since librosa 0.9 requires SciPy 1.2 or newer.

Differential Revision: [D34386898](https://our.internmc.facebook.com/intern/diff/D34386898)

### CODE REVIEW

**Comment by peterbell10** [comment]
Not sure what's going on with the tests here. The code that's failing doesn't have anything to do with librosa.

File: .circleci/docker/common/install_conda.sh [hunk 1]
```diff
@@ -96,26 +94,27 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   conda_install nnpack -c killeent
 
   # Install some other packages, including those needed for Python test reporting
-  # TODO: Why is scipy pinned
+  # Pin SciPy because of failing distribution tests (see #60347)
```
Comment (peterbell10) [inline .circleci/docker/common/install_conda.sh:97]: Test failures were due to `pip install librosa` also upgrading SciPy, which in turn caused #60347 to re-appear. Workaround for now is to update the SciPy pin.

**Review by mruberry — APPROVED** [review]
Awesome! Thanks @peterbell10!

**Comment by mruberry** [comment]
@mruberry has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34386898).

**Pull request closed without merging**